lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -411,6 +421,26 @@ private boolean isHmrcInheritedIdentityWithStrongerVotPresent(
         }
     }
 
+    private void sendInheritedIdentityReceivedAuditEvent(
+            SignedJWT signedInheritedIdentityJWT, AuditEventUser auditEventUser)
+            throws SqsException, ParseException, AuditExtensionException, CredentialParseException,
+                    UnrecognisedVotException {
+        try {
+            AuditExtensionsVcEvidence auditExtensions =
+                    getExtensionsForAudit(signedInheritedIdentityJWT, null);
+            auditService.sendAuditEvent(
+                    new AuditEvent(
+                            AuditEventTypes.IPV_INHERITED_IDENTITY_VC_RECEIVED,
+                            configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
+                            auditEventUser,
+                            auditExtensions));
+        } catch (ParseException | IllegalArgumentException e) {
+            throw new CredentialParseException(
+                    "Encountered a parsing error while attempting to parse or compare credentials",
```

Comment:
Think this error message needs an update. e.g. we're not comparing credentials anymore